influxdb/server.sls

```diff
@@ -35,6 +35,27 @@ influxdb_default:
     - pkg: influxdb_packages
 {%- endif %}
 
+{{server.data.dir}}:
```

Comment:
Not sure all this code is necessary as from my perspective this should be handled by InfluxDB packaging